components/vf-banner/README.md

```diff
@@ -2,6 +2,14 @@
 
 [![npm version](https://badge.fury.io/js/%40visual-framework%2Fvf-banner.svg)](https://badge.fury.io/js/%40visual-framework%2Fvf-banner)
 
+## Usage Guidelines
+
+Informs visitors about important changes or persistent conditions. Use this component if you need to communicate to visitors in a prominent way. Banners are placed at the top of the page or section they apply to, and below the page or section header.
+
+### Exceptions
+
+The GDPR Banner, is the `--fixed` variant which needs to be 'sticky' to the bottom of the browser viewport
```

Comment:
Is this an exception to the rule, seems consistent with the above?

Reply:
I think you might be right here. I’ll amend.